.gitignore

```diff
@@ -13,9 +13,16 @@ __pycache__/
 TODO
 node_modules
 
+# Pipenv
+Pipfile
+Pipfile.lock
+
 # IDE and Tooling files
 .idea/*
 *~
 
 # macOS
 .DS_Store
+
+# VSCode Tooling files
+.vscode/
```

Comment:
Please add these to your system-local gitignore/gitexclude rather than the project one.

Reply:
You mean just `.vscode/` folder or also `Pipfile` and `Pipfile.lock`? I definitely don't want my other `Pipfile` and `Pipfile.lock` files in `.gitignore` :) but I can workaround it if needed.

Reply:
Just the vscode one - it's not part of the Python ecosystem directly, and if we had a folder for every editor it would get messy!

Reply:
Yeah makes sense.
And what about leaving `Pipfile` and `Pipfile.lock` in the repo? It shouldn't affect anyone except for those using Pipenv and for those using it it would make things just easier, don't you think? I can add some section to the Contribution guide how to setup repo using PIpenv or I can move it to another PR.

Reply:
Pipfiles only make sense on end projects, not on libraries, so they don't really belong in this repo.